CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+# Framework Philosophies
+
+This document outlines the goals, ideas, and roadmap for the rg-bot library (as well as our public-facing libraries in general). This is a living document, and we encourage players to participate and contribute to this document.
+
+## Core Facts
+
+The success of this library is dictated by various core facts that should be true. Our goal is to make sure that these facts are always true, and any change that causes one of these facts to not be true should be re-evaluated.
```

Comment:
Sounds a bit wonky because facts are implicitly true. Perhaps something along the lines of 

"The success of this library is dictated by its adherence to several core facts. Our goal is to make sure that these facts inform changes to the library. Any change that violates one or more of these facts must be re-evaluated."

Reply:
Fixed!